### ISSUE TITLE
Doesn't build on GHC 7.10 with recent dependencies

### ISSUE THREAD

**Issue by radix** [travitch/persistent-vector#1]
Trying to build persistent-vector gives me this:

```
src/Data/Vector/Persistent/Array.hs:55:7:
    Ambiguous occurrence ‘traverse’
    It could refer to either ‘Data.Vector.Persistent.Array.traverse’,
                             defined at src/Data/Vector/Persistent/Array.hs:465:1
                          or ‘P.traverse’,
                             imported from ‘Prelude’ at src/Data/Vector/Persistent/Array.hs:67:1-56
                             (and originally defined in ‘Data.Traversable’)
```

Hacking around that problem (in a way I'm not sure is backwards compatible), I then get this trying to build the benchmark:

```
bench/pvBench.hs:3:8:
    Could not find module ‘Criterion.Config’
    Perhaps you meant
      Criterion.Monad (from criterion-1.1.0.0@crite_IAvDJexKd482tiz7RXZaU5)
    Use -v to see a list of the files searched for.
```

after simply deleting that import we run into a real incompatibility with Criterion's `defaultMainWith` -- apparently they got rid of the preparation (`:: Criterion ()`) argument. I don't know anything about criterion so I'm not sure how to fix this one.

**Comment by travitch** [comment]
Thanks for the report - I'll fix that shortly

**Comment by travitch** [comment]
Should be fixed in 9d14c6ba3f5e3601c00 and version 0.1.1 on hackage

**Comment by radix** [comment]
Thanks @travitch! :sunglasses:
